### PR TITLE
Deprecate Python modules GEXFIO and GraphMLIO

### PR DESCRIPTION
**Pull request by fabratu**
The functionality of both modules is moved to `graphio.pyx`.

### CODE REVIEW

File: networkit/GEXFIO.py [hunk 1]
```diff
@@ -1,482 +1,38 @@
-import queue
-import xml.etree.cElementTree as ET
-from xml.dom import minidom
-from .graph import Graph
-from .dynamics import GraphEvent
+from warnings import warn
+
+from .graphio import GEXFReader as ioGEXFReader
+from .graphio import GEXFWriter as ioGEXFWriter
```
Comment (avdgrinten) [inline networkit/GEXFIO.py:4]: Classes should use `CamelCase`.

File: networkit/GraphMLIO.py [hunk 2]
```diff
@@ -1,287 +1,54 @@
-import xml.etree.cElementTree as ET
-import xml.sax
+from warnings import warn
 
-from .graph import Graph
+from .graphio import GraphMLSAX as ioGraphMLSAX
+from .graphio import GraphMLReader as ioGraphMLReader
+from .graphio import GraphMLWriter as ioGraphMLWriter
```
Comment (avdgrinten) [inline networkit/GraphMLIO.py:5]: Likewise.

**Review by avdgrinten — COMMENTED** [review]
LGTM except for nitpicks below.

**Review by avdgrinten — APPROVED** [review]

**Comment by fabratu** [comment]
Had to rebase due to a merge conflict.

**Review by avdgrinten — APPROVED** [review]

**Pull request merged**